src/duqtools/ids/_copy.py

```diff
@@ -51,6 +52,45 @@ def get_imas_ual_version():
     return imas_version, ual_version
 
 
+def add_provenance_info(ids: ImasHandle):
+    """Add provenance information to ids.
+
+    Parameters
+    ----------
+    ids:
+        handle to add provenance information to
```

Comment:
```suggestion
    ids: ImasHandle
        Handle to add provenance information to.
```

Reply:
Ah, yes I didn't have the merge.py file in this branch yet, and didn't see sync being used anywhere, but it definitely makes sense to add it there